### PR TITLE
Feature/faster model menu

### PR DESCRIPTION
**Pull request by solarisstar**

### CODE REVIEW

File: code/ui/ui_playermodel.c [hunk 1]
```diff
@@ -11,6 +11,13 @@
 //=================================================
 #include "ui_local.h"
 
+#define START_PROFILING()     clock_t start = clock(); \
```
Comment (MJLHThomassen) [inline code/ui/ui_playermodel.c]: These should be removed

File: code/ui/ui_playermodel.c [hunk 2]
```diff
@@ -1816,7 +1831,7 @@ static void PlayerModel_MenuInit(int menuFrom)
     s_playermodel.model.textcolor = CT_BLACK;
     s_playermodel.model.textcolor2 = CT_WHITE;
 
-    //y =	88;
+    //y =   88;
```
Comment (MJLHThomassen) [inline code/ui/ui_playermodel.c:1813]: What does this comment mean?

File: code/ui/ui_playermodel.c (hunk 1 above)
Comment (solarisstar) [inline code/ui/ui_playermodel.c]: Totally right. Not quite finished with this branch yet.

File: code/ui/ui_playermodel.c (hunk 2 above)
Comment (solarisstar) [inline code/ui/ui_playermodel.c:1813]: It's some of the comment litter left in the project.

**Comment by solarisstar** [comment]
Not quite done with this branch yet. I've pushed the model loading to startup, but I want to see if I can make ti faster.

**Comment by solarisstar** [comment]
Finished. Let me know if you have any comments

File: code/game/g_turrets.c [hunk 3]
```diff
@@ -880,7 +880,11 @@ void SP_misc_turret(gentity_t *base)
     arm->lastEnemy = head;
     head->r.ownerNum = arm->s.number;
     arm->activator = head->activator = base;
+<<<<<<< HEAD
```
Comment (MJLHThomassen) [inline code/game/g_turrets.c]: Uhm...

File: code/ui/ui_playermodel.c [hunk 4]
```diff
@@ -1140,8 +1178,13 @@ static void PlayerModel_BuildList(void)
 
     ///Com_Printf("%i folders found\n", numdirs );
 
+    int pool_idx = 0;
+    ht_element_t  ele_pool[MAX_PLAYERCHARS];
```
Comment (MJLHThomassen) [inline code/ui/ui_playermodel.c]: i would prefer some more descriptive variable names. We're not living in the 80's anymore where with 640x480 horizontal screen space came at a premium :p

File: code/game/g_turrets.c (hunk 3 above)
Comment (solarisstar) [inline code/game/g_turrets.c]: my bad, serves me right for working on an airplane

**Pull request merged**